pkg/cmd/label/shared.go

```diff
@@ -1,7 +1,51 @@
 package label
 
+import (
+	"reflect"
+	"strings"
+	"time"
+)
+
+var labelFields = []string{
+	"color",
+	"createdAt",
+	"description",
+	"id",
+	"isDefault",
+	"name",
+	"updatedAt",
+	"url",
+}
+
 type label struct {
-	Name        string `json:"name"`
-	Color       string `json:"color"`
-	Description string `json:"description,omitempty"`
```

Comment:
FWIW, I only added annotations to these because they are the only ones we ever use to serialize data. The rest we only use to deserialize so the annotations are extraneous. Obviously doesn't hurt. 🙂